### PR TITLE
Remove lamp version of Status component

### PR DESCRIPTION
**Pull request by bryancunningham-okta**
Move to only having 'pill' variant of Status component

[OKTA-672550](https://oktainc.atlassian.net/browse/OKTA-672550)

## Testing & Screenshots

- [x] I have confirmed this change with my designer and the Odyssey Design Team.

### CODE REVIEW

File: packages/odyssey-react-mui/src/Status.tsx [hunk 1]
```diff
@@ -36,14 +36,13 @@ export type StatusProps = {
    * The style of the Status indicator
    */
   variant?: (typeof statusVariantValues)[number];
+  /**
+   * @deprecated All Statuses are now `pill` variant.
+   */
+  /** **Deprecated:** All Statuses are now `pill` variant */
```
Comment (chrispulsinelli-okta) [inline packages/odyssey-react-mui/src/Status.tsx:42]: Curious if you meant to put this comment block after the props and just above the component variable declaration below?

**Review by chrispulsinelli-okta — APPROVED** [review]
This looks good to me - but left a comment for you to check out.

File: packages/odyssey-react-mui/src/Status.tsx (hunk 1 above)
Comment (bryancunningham-okta) [reply packages/odyssey-react-mui/src/Status.tsx:42]: Whoops, yeah, this needs to be above `variant`. I'll fix. Thanks!

**Comment by bryancunningham-okta** [comment]
Closing this. Odyssey team decided not to pursue this update

**Pull request closed without merging**